### PR TITLE
Adding setup script

### PR DESCRIPTION
**Pull request by ViktorBarzin**
Setup loki with 3 commands and spend 5 min rather with 20+ commands and 1 hour googling

Read setup-steps.txt for more info

### CODE REVIEW

File: setup-loki.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
+mkvirtualenv -p /usr/bin/python3 loki
```
Comment (kdkocev) [inline setup-loki.sh]: It is a good practice to keep all virtualenvs in a single folder. In HackBulgaria we have all the envs in `~/code/env/` so you should probably remake `loki` to `~/code/env/loki` or something else using a non-relative path.

File: setup-loki.sh [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+git clone https://github.com/HackSoftware/Loki.git
```
Comment (kdkocev) [inline setup-loki.sh]: Shouldn't you already have Loki cloned if you're running this script? I mean, if it's going to be added in Loki, you would've already cloned Loki in order to run the script

File: setup-steps.txt [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+sudo chmod +x setup-loki.sh
+
+./install-libs.sh
+# Must be run with source or virtualenv breaks !!!
+source setup-libs.sh
+
+# Go to Loki/ and run "python manage.py runserver"
+# Should be up with all the jquery stuff
+
+# If something does not work, repeat the proccess as root
```
Comment (kdkocev) [inline setup-steps.txt]: This file is in `.txt` format. If you want it to be executable - make it a `.sh` if you want it to serve as a documentation - make it a github Markdown

File: setup-loki.sh (hunk 1 above)
Comment (ViktorBarzin) [inline setup-loki.sh]: `mkvritualenv` is a wrapper around `virtualenv` which stores all enviorments at  `~/.virtualenvs/`
If you want to specify another folder, you can set the `WORKON_HOME` variable to whatever folder you'd like.

The path I have added is the path to the python3 binary. By default `mkvirtualenv` uses the system default python which for most distros is python2.

File: setup-loki.sh (hunk 2 above)
Comment (ViktorBarzin) [inline setup-loki.sh]: 1 line less. I agree

**Pull request closed without merging**